### PR TITLE
Remove unused tests

### PR DESCRIPTION
**Pull request by jonnii**
From what I understand the concept of a "virtual" list-view-item
is going away what with Glimmer (tm) making it unnecessary.

### CODE REVIEW

**Comment by jonnii** [comment]
@stefanpenner what are your thoughts on this?

**Comment by mmun** [comment]
We have some plans on reworking these in new context. I think @raytiley will take the reigns on it.

@jonnii I think it would be useful to cut up the acceptance tests and see if any are already passing.

**Comment by jonnii** [comment]
@mmun i suppose you could rewrite these but they would be a complete re-write as i think most of this stuff is gone. the value here (i think) is the tests which scroll around, but i think those are also covered by the `scroll-top-test.js`.

**Comment by mmun** [comment]
Which tests are you talking about? I was referring to the acceptance tests in the tests/acceptance/list-view-test.js file.

**Comment by jonnii** [comment]
@mmun just these ones, the recycling-tests. from what i understand all the hackery put into ember-list-view to manage recycling views just comes for free now with glimmer.

**Comment by jonnii** [comment]
i'm sure most of the other tests can and will be re-written, but these ones deal with `ListItemView` and `ReusableListItemView` which are long gone.

**Comment by mmun** [comment]
Ok let me rephrase: I think @krisselden and @raytiley have an idea for how to recycle the recycling-tests.

I was suggesting that it would be helpful if you dug through the list-view-test.js next.

**Comment by jonnii** [comment]
ah, ok. i can do that! will leave this here for now.

**Comment by mmun** [comment]
:+1:

**Comment by shaunc** [comment]
the multiheight test could usefully be converted into a test of the mixed grid layout which is currently untested.
